android/src/main/java/com/swmansion/rnscreens/ScreenStack.java

```diff
@@ -169,65 +169,63 @@ protected void performUpdate() {
       }
     }
 
-    boolean customAnimation = false;
+    // variables used for "open/back animation" logic
```

Comment:
```suggestion
    // variables used for "open/close animation" logic
```

Reply:
Fixed in 7e8721b08db4ceda2942dc73f00470caff78edd8